docs/en/developers/04_Pull_requests.md

```diff
@@ -0,0 +1,71 @@
+# Opening a pull request
+
+So you want to propose a patch to the community? It's time to open a [pull request](https://github.com/FreshRSS/FreshRSS/pulls)!
+
+When you'll open it, your message will be prefilled with a message based on [a template](https://github.com/FreshRSS/FreshRSS/blob/master/docs/pull_request_template.md). It contains a checklist to make sure you didn't forget anything. It is very important to verify you did everything mentioned so documentation is up-to-date, the commit history stays clear and the code is always stable.
```

Comment:
It might be a bit confusing to have a checkbox for unit tests, given that we have almost no existing test. However, writing how the code was tested is useful (e.g. on which platform, browser, PHP version, database...)

Reply:
The point is to encourage to write more unit tests, so I think the checkbox is pertinent. However, we can add an "optional" label if testing the feature is too hard or really time-consuming. I'll add a section to let people explain how to test the feature manually :+1: